### PR TITLE
fix interactive add route skip functions

### PR DESCRIPTION
**Pull request by ashleywang1**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/470

### CODE REVIEW

File: projects/gloo/cli/pkg/cmd/add/route.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func pluginsFromInput(input options.RoutePlugins) (*v1.RoutePlugins, error) {
 
 func destSpecFromInput(input options.DestinationSpec) (*v1.DestinationSpec, error) {
 	switch {
-	case input.Aws.LogicalName != "":
```
Comment (EItanya) [inline projects/gloo/cli/pkg/cmd/add/route.go]: `case  input.Aws.LogicalName = surveyutils.NoneOfTheAbove`?
just to make it easier to read

Comment (ashleywang1) [reply projects/gloo/cli/pkg/cmd/add/route.go]: That part was changed to `case input.Aws.LogicalName != "" && input.Aws.LogicalName != surveyutils.NoneOfTheAbove:`, so this runs when "none of the above" was selected or nothing was selected / the question wasn't asked.

Comment (EItanya) [reply projects/gloo/cli/pkg/cmd/add/route.go]: Ahhh yes,  I misread the code

**Pull request merged**